README.md

```diff
@@ -1,16 +0,0 @@
-# uwb-hacks [![Build Status](https://travis-ci.com/UWB-ACM/uwb-hacks.svg?branch=master)](https://travis-ci.com/UWB-ACM/uwb-hacks)
```

Comment:
Same thing applies with the readme. This should be updated to include instructions for the new website